### ISSUE TITLE
Should JOPA support automatic conversion to string?

### ISSUE THREAD

**Issue by ledsoft** [kbss-cvut/jopa#43]
@psiotwo 
When an attribute is mapped to `OWLDataProperty` or `OWLAnnotationProperty` and the field type is `String` (or a collection of strings), should JOPA perform automatic conversion of values loaded from ontology to it, regardless of their actual type? For example:

```Java
class A {
@OWLAnnotationProperty(iri="dcterms:created")
private String created;
}
```

and the following triple:
```
ex:a dcterms:created "2018-10-28T23:59:59"^^xsd:dateTime .
```
Should it be loaded and automatically converted to `String`, or should it be skipped because its value type does not correspond to the field range? Or, should this be configurable?
Note that on update/insertion to repository, the triple value would be based on the attribute mapping, i.e., always string. So the dateTime-typed value would be replaced with a string in the above example on update.

It seems like a reasonable feature to support.

**Comment by psiotwo** [comment]
A systematic way would be to support for each OWL(Datatype|Annotation)Property (whichever the type of the Java field is) the possibility to be expressed in Java as the **lexical form** of the RDF literal. 

This option should be configurable on the given property. One of the reasons is that String literals with language tags have the same lexical form as String literals without language tags - and these two would not be distinguishable in JOPA, e.g. "person" and "person"@en are both mapped to the String field with value "person".

**Comment by ledsoft** [comment]
Closing this one, as it has been resolved by #50.
